peripherals/sai/sai.yaml

```diff
@@ -0,0 +1,173 @@
+"SAI,SAI?":
+  "*CR1":
+    #MCKDIV:
+    NODIV:
+      MasterClock: [0, "Master clock generator is enabled"]
+      NoDiv: [1, "No divider used in the clock generator (in this case Master Clock Divider bit has no effect)"]
+    DMAEN:
+      Disabled: [0, "DMA disabled"]
+      Enabled: [1, "DMA enabled"]
+    SAIEN:
+      Disabled: [0, "SAI audio block disabled"]
+      Enabled: [1, "SAI audio block enabled"]
+    OUTDRIV:
+      OnStart: [0, "Audio block output driven when SAIEN is set"]
+      Immediately: [1, "Audio block output driven immediately after the setting of this bit"]
+    MONO:
+      Stereo: [0, "Stereo mode"]
+      Mono: [1, "Mono mode"]
+    SYNCEN:
+      Asynchronous: [0, "audio sub-block in asynchronous mode"]
+      Internal: [1, "audio sub-block is synchronous with the other internal audio sub-block. In this case, the audio sub-block must be configured in slave mode"]
+      External: [2, "audio sub-block is synchronous with an external SAI embedded peripheral. In this case the audio sub-block should be configured in Slave mode"]
+    CKSTR:
+      RisingEdge: [0, "ignals generated by the SAI change on SCK rising edge, while signals received by the SAI are sampled on the SCK falling edge"]
+      FallingEdge: [1, "Signals generated by the SAI change on SCK falling edge, while signals received by the SAI are sampled on the SCK rising edge"]
```

Comment:
These values are a bit confusing - RM0090 talks about 0 being "data strobing edge is falling edge of SCK". Did another RM have a clearer explanation?

Also, missing the first letter S in Signals.